### DISCUSSION TITLE
Can someone given me an idea about the Matern Kernel in general? I need some sort of idea regarding how to visualize it and understand it's basics

### DISCUSSION THREAD

**Discussion by Pranav110500** [GPflow/GPflow#1937] [General]
So, I am working on some geospatial data and am using the GPflow framework to create a model for Gaussian Process Regression, along with a set of kernels are given which can be implemented, I just wanted to get some basic idea behind how this kernel works, so any sort of basic theory/descriptions about it will be really helpful.

**Comment by st--** [comment]
Hi @Pranav110500 , that's a fairly vague question:) you might find something like [this blogpost](https://andrewcharlesjones.github.io/journal/matern-kernels.html) helpful (I had a quick check that it seems sensible).
